packages/rest/src/Fetcher.ts

```diff
@@ -179,7 +179,7 @@ export class Fetcher {
     }
 
     if (response.ok) {
-      response.status === 204 ? undefined : entry.resolve(await response.json());
+      response.status === 204 ? entry.resolve() : entry.resolve(await response.json());
```

Comment:
can this be changed to calling `entry.resolve` with the ternary inside of it. i'm honestly a bit embarrased that the original code as a top level ternary and dont want it to stay that way.

Reply:
like `entry.resolve(response.status === 204 ? void 0 : await response.json())` ?